source/api/commands/server.md

```diff
@@ -67,7 +65,7 @@ Option | Default | Description
 
 ***After starting a server:***
 
-- Any request that does not match a {% url `cy.route()` route %} will be sent a `404` status code.
+- Any request that does **NOT** match a {% url `cy.route()` route %} will {% url 'pass through to the server' network-requests#Don’t-Stub-Responses %}.
```

Comment:
The meat of this PR:

* document the correct behaviour
* hyperlink to the relevant section of the _Network Requests_ section to be helpful.